src/middleware/organisations.middleware.js

```diff
@@ -2,7 +2,14 @@ import { logPageError } from './common.middleware.js'
 import { fetchMany, renderTemplate } from './middleware.builders.js'
 
 const fetchOrganisations = fetchMany({
-  query: ({ req, params }) => 'select name, organisation from organisation',
+  query: ({ req, params }) => {
+    return `
+      select name, organisation
+      from organisation
+      WHERE organisation like '%local-authority%' OR organisation like '%national-park-authority%'
```

Comment:
```suggestion
      WHERE organisation like 'local-authority:%' OR organisation like 'national-park-authority:%'
```
I say drop the wildcard before the string and also add the `:` so it does an exact match on the prefix

Reply:
Also should we filter by end date because it looks like some have ended https://datasette.planning.data.gov.uk/digital-land/organisation?_sort=organisation&organisation__startswith=local-authority%3A